tests/e2e/Services/Databases/DatabasesCustomClientTest.php

```diff
@@ -225,4 +225,95 @@ public function testUpdateWithoutPermission(): array
 
         return [];
     }
+
+    public function testUpdateTwoWayRelationship(): void
+    {
+
+        $database = $this->client->call(Client::METHOD_POST, '/databases', [
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ], [
+            'databaseId' => ID::unique(),
+            'name' => 'Test Database'
+        ]);
+
+        $databaseId = $database['body']['$id'];
+
+
+        //creating collection 1
```

Comment:
Always try to match conventions and styles that already exist in a codebase. 

```suggestion
        // Creating collection 1
```